src/sourcemaps/tools/sentry-cli.ts

```diff
@@ -126,19 +132,125 @@ async function createAndAddNpmScript(
   } ${relativePosixArtifactPath}`;
 
   packageDotJson.scripts = packageDotJson.scripts || {};
-  packageDotJson.scripts[NPM_SCRIPT_NAME] = sentryCliNpmScript;
+  packageDotJson.scripts[SENTRY_NPM_SCRIPT_NAME] = sentryCliNpmScript;
 
   await fs.promises.writeFile(
     path.join(process.cwd(), 'package.json'),
     JSON.stringify(packageDotJson, null, 2),
   );
 
   clack.log.info(
-    `Added a ${chalk.cyan(NPM_SCRIPT_NAME)} script to your ${chalk.cyan(
+    `Added a ${chalk.cyan(SENTRY_NPM_SCRIPT_NAME)} script to your ${chalk.cyan(
       'package.json',
-    )}. Make sure to run this script ${chalk.bold(
-      'after',
-    )} building your application but ${chalk.bold('before')} deploying!`,
+    )}.`,
+  );
+}
+
+async function askShouldAddToBuildCommand(packageDotJson: PackageDotJson) {
+  const shouldAddToBuildCommand = await abortIfCancelled(
+    clack.select({
+      message: `Do you want to automatically run the ${chalk.cyan(
+        SENTRY_NPM_SCRIPT_NAME,
+      )} script after each production build?`,
+      options: [
+        {
+          label: 'Yes',
+          value: true,
+          hint: 'This will modify your prod build comamnd',
+        },
+        { label: 'No', value: false },
+      ],
+      initialValue: true,
+    }),
+  );
+
+  Sentry.setTag('modify-build-command', shouldAddToBuildCommand);
+
+  if (shouldAddToBuildCommand) {
+    await traceStep('sentry-cli-add-to-build-cmd', () =>
+      addSentryCommandToBuildCommand(packageDotJson),
+    );
+  } else {
+    clack.log.info(
+      `No problem, just make sure to run this script ${chalk.bold(
+        'after',
+      )} building your application but ${chalk.bold('before')} deploying!`,
+    );
+  }
+}
+
+/**
+ * Add the sentry:sourcemaps command to the prod build command in the package.json
+ * - Detect the user's build command
+ * - Append the sentry:sourcemaps command to it
+ *
+ * @param packageDotJson The package.json which will be modified.
+ */
+async function addSentryCommandToBuildCommand(
+  packageDotJson: PackageDotJson,
+): Promise<void> {
+  // This usually shouldn't happen because earlier we added the
+  // SENTRY_NPM_SCRIPT_NAME script but just to be sure
+  packageDotJson.scripts = packageDotJson.scripts || {};
+
+  let buildCommand = packageDotJson.scripts.build;
+  let isProdBuildCommand = false;
+
+  const pacMan = detectPackageManager() || 'npm';
+
+  if (buildCommand) {
+    isProdBuildCommand = await abortIfCancelled(
+      clack.confirm({
+        message: `Is ${chalk.cyan(
+          `${pacMan} run build`,
+        )} your production build command?`,
+      }),
+    );
+  }
```

Comment:
Right now, I only checked for `build` but we could establish an ordered list of other heuristics, such as `build:prod`, `prod`, etc. Not sure if we wanna go down this route though given that the possibilities are endless...

Reply:
What about:

1. Check for `build` exact match
2. If no exact match, check for first entry that `script.includes('build')`

This is obv. not perfect but maybe a 90% good enough heuristic? IMHO it's also fine to only auto-select `build` and have no auto-selection otherwise, though...

Reply:
Yeah, let's do this - good suggestion!